classes/Cron.php

```diff
@@ -54,7 +55,8 @@ public static function route()
     public static function processQueue()
     {
         // limit to one process by only proceeding if no (or an expired left over) lockfile exists
-        $lockfile = kirby()->root('content') . DS . '.commentions_queuelock';
+        $logfolder = kirby()->root('site') . DS . 'logs' . DS . 'commentions';
+        $lockfile = $logfolder . DS . 'queuelock.txt';
```

Comment:
PHP accepts both `\\` and `/` as directory separators under all OSs, so you can also safely use `/` instead of the `DS` constant. This is clearly opinionated, but I personally prefer `"{$logfolder}/queuelock.txt"` over concetanating everything with dots.